probe-rs/src/debug/mod.rs

```diff
@@ -211,6 +211,7 @@ pub struct StackFrameIterator<'debuginfo, 'probe, 'core> {
     pc: Option<u64>,
     registers: Registers,
     inlining_state: InlineFunctionState,
+    static_variables: Vec<Variable>,
```

Comment:
Why is this part of the `StackFrameIterator`? Do the static variable depended on the stack frames in any way? If not, I would prefer if it is separate.

Reply:
It does NOT depend on the StackFrameIterator. It belongs close to each of the stack frames, as it kinda belongs to 'all' stack frames, and it did not make sense to duplicate per frame.   What would be your suggestion?

Reply:
As mentioned in the other comment, I would just remove it from `StackFrameIterator`, and get it directly from `DebugInfo`.